src/components/MessageInput/MessageInput.tsx

```diff
@@ -473,49 +482,20 @@ export const MessageInputWithContext = <
             <Input
               additionalTextInputProps={additionalTextInputContainerProps}
```

Comment:
Could toss the other open close command props on here too I suppose, not a big deal though.